src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -101,6 +101,13 @@ sylius_product:
                     form:
                         default: Sylius\Bundle\CoreBundle\Form\Type\ProductTranslationType
 
+sylius_association:
+    resources:
+        product:
+            subject: Sylius\Component\Core\Model\Product
```

Comment:
It would stop working if someone overrides product class and forgets to change it here, bad DX.